src/main.js

```diff
@@ -61,6 +61,11 @@ const filterBeforeCreate = (toast, toasts) => {
   return toast;
 };
 
+// Cypress automatically sets window.Cypress by default
+if (window.Cypress) {
+  window.__store__ = store;
```

Comment:
it's not clear to me what this attribute means and why we need to set this variable when cypress is being used

Reply:
This is just to add the vuex store into the global namespace -- as it's used by commands.js file

Reply:
Yes, this is the vuex store. We need it within Cypress also, hence we're registering the `store` with the global `window` object

Reply:
My doubt was also regarding why are we using `window.__store__` instead of, say, `window.store`? Is there a convention?

Reply:
There's no convention. I believe it's mostly to avoid conflict with existing functionality and other packages. `window.store` can be initialized or used by some other functionalities. A different name gives a bit more flexibility and chances of avoiding conflicts.